src/plugins/intel_cpu/src/nodes/executors/fullyconnected_implementations.cpp

```diff
@@ -301,6 +316,36 @@ const std::vector<ExecutorImplementation<FCAttrs>>& getImplementations() {
                     context,
                     false);
             })
+        OV_CPU_INSTANCE_ACL(
+            "fullyconnected_acl",
+            ExecutorType::Acl,
+            OperationType::FullyConnected,
+            ShapeTolerance::Agnostic,
+            // supports
+            [](const FCConfig& config) -> bool {
```

Comment:
let's ensure the tests are passed and disable the executor for now.
There is no rush to enable it and replace with the oneDNN one.
We need to make sure we don't have degradations first.

Reply:
@EgorDuplensky I'll disable it when review will be ended